src/Workspaces/Core/Portable/Workspace/DocumentActiveContextChangedEventArgs.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    public sealed class DocumentActiveContextChangedEventArgs : EventArgs
+    {
+        public Workspace Workspace { get; }
+        public SourceTextContainer SourceTextContainer { get; }
+        public DocumentId PreviousContextDocumentId { get; }
+        public DocumentId NewContextDocumentId { get; }
```

Comment:
Copy/pasting from my old comment:

@jaredpar Let me start by saying that I'm happy to change it to Old / New if that's what we decide to go with, but I'd like to share my thought process.

In the other Old/New cases (that I've seen so far), there's actually an old version of something and a new version of that same thing. In this case, nothing changed in quite the same way. There's not some tangible entity whose DocumentId changed from one thing to another. One of them isn't "old" in the sense that it's out-of-date or from an earlier snapshot, just that it was the previous active context document.

Having said that, there's no precedent (that I know of) for this new naming scheme, so if I somehow convinced you that straying from Old/New was okay, we'd still want to choose our new words carefully.

Again, I'm happy to switch it to Old/New, but let me know if my argument even begins to make you think twice about it. :smile: